component/attr_cache/attr_cache.go

```diff
@@ -525,12 +473,10 @@ func (ac *AttrCache) cacheAttributes(pathList []*internal.ObjAttr) {
 		ac.cacheLock.Lock()
 		defer ac.cacheLock.Unlock()
 		for _, attr := range pathList {
-			// TODO: will this cause a bug when cacheDirs is enabled?
-			if len(ac.cacheMap) > ac.maxFiles {
-				log.Debug("AttrCache::cacheAttributes : %s skipping adding path to attribute cache because it is full", pathList)
-				break
-			}
-			ac.cacheMap[internal.TruncateDirName(attr.Path)] = newAttrCacheItem(attr, true, currTime)
+
+			// TODO: this will require a tree traversal / scan to get cachedItems count
```

Comment:
I don't understand this comment. If it's outdated, let's drop it.
Also, the two blank lines surrounding this insert call seem unnecessary.